packages/cxl-lumo-styles/scss/themes/vaadin-horizontal-layout.scss

```diff
@@ -5,3 +5,12 @@
     color: var(--lumo-primary-color);
   }
 }
+
+:host([theme~="cxl-marketing-hero-details"]) {
+  font-size: var(--lumo-font-size-s);
+
+  ::slotted(span) {
+    display: inline-flex;
+    align-items: center;
```

Comment:
So `align-items: center` **is** needed in Chrome?

Reply:
It is needed

Reply:
No.

See [Screencast from 2023-02-23 13-48-31.webm](https://user-images.githubusercontent.com/147228/220897731-06178f60-3efc-4764-8b78-ba4f01a65522.webm)

I checked at https://deploy-preview-263--conversionxl-aybolit.netlify.app/?path=/story/cxl-ui-cxl-app-layout--cxl-app-layout-1-cc

Both Chrome and Firefox behave exactly the same, so it's not about the browser.

`align-items: center` causes the icon to align itself with bottom of "drop" letters like p, but this mis-aligns it with the capital letter the icon is actually closest to.

Unless you're seeing something markedly different, let's go without `align-items: center`.

Reply:
> `align-items: center` causes the icon to align itself with bottom of "drop" letters like p, but this mis-aligns it with the capital letter the icon is actually closest to.
> Unless you're seeing something markedly different, let's go without `align-items: center`.

I see the same and I like it. I think icon should be in the middle of the whole text.

**GOOD** - with align-items
![Screenshot 2023-02-23 at 13 02 49](https://user-images.githubusercontent.com/5165721/220900544-83ba066a-6de1-4454-a87f-c7a2882ca01b.png)

**BAD** - without align-items
![Screenshot 2023-02-23 at 13 03 01](https://user-images.githubusercontent.com/5165721/220900579-e9a49847-ab6c-41db-9822-6584499615c6.png)